build.sbt

```diff
@@ -27,12 +27,14 @@ lazy val commonSettings = projectSettings ++ coverageSettings ++ compilerSetting
 lazy val casper = (project in file("casper"))
   .settings(commonSettings: _*)
   .settings(
-    libraryDependencies ++= commonDependencies ++ protobufDependencies,
+    libraryDependencies ++= commonDependencies ++ protobufDependencies ++ Seq(
+      catsCore,
```

Comment:
Cats no longer needed after refactor to use Iterator instead of State monad

Reply:
Reverted